docs/configuration/actions.md

```diff
@@ -123,43 +124,108 @@ actions:
   - bash -c "sed -i -r \"s/Release:(\s*)\S+/Release:\1${PACKIT_RPMSPEC_RELEASE}%{?dist}/\" my_specfile_path"
 ```
 
+### `commit-message`
+
+Our internal API differentiates between the commit title and description, so we
+have an additional requirement on separating the commit title by an empty line.
+You can, of course, provide *just* the commit title.
+
+#### Debugging
+
+For your own debugging purposes we allow arbitrary output before outputting the
+actual commit message. In such case, your commit message **needs** to be
+separated by the following separator (on a separate line):
+
+    ---%<--- snip ---%<--- here ---%<---
+
+:::tip
+
+This separator is exposed as an environment variable `PACKIT_DEBUG_DIVIDER`.
+
+:::
+
+#### Examples
+
+<details>
+<summary>Example of correct output</summary>
+
+    debug output
+    ---%<--- snip ---%<--- here ---%<---
+    Rebase to new upstream release 0.42.69
+
+    - Resolves rhbz#1234
+
+This output can be produced by the following config:
+```yaml
+actions:
+  commit-message:
+    - echo 'debug output'
+    - bash -c 'echo ${PACKIT_DEBUG_DIVIDER}'
+    - bash -c 'echo -e "Rebase to new upstream release ${PACKIT_PROJECT_VERSION}\n"'
+    - echo '- Resolves rhbz#1234'
+```
+
+</details>
+
+<details>
+<summary>Example of incorrect output</summary>
+
+    ---%<--- snip ---%<--- here ---%<---
+
+
+    missing the commit title
+
+</details>
+
 ## Environment variables set by packit
 
 Additionally, packit sets several environment variables for the actions:
 
-`PACKIT_UPSTREAM_PROJECT_NAME` — set to the `upstream_package_name` value, if any  
-`PACKIT_DOWNSTREAM_PROJECT_NAME` — set to the `downstream_package_name` value, if any  
-`PACKIT_CONFIG_PACKAGE_NAME` — set to the package name key for the `packages` dictionary in a monorepo project,
+* `PACKIT_UPSTREAM_PROJECT_NAME` — set to the `upstream_package_name` value, if any
+* `PACKIT_DOWNSTREAM_PROJECT_NAME` — set to the `downstream_package_name` value, if any
+* `PACKIT_CONFIG_PACKAGE_NAME` — set to the package name key for the `packages` dictionary in a monorepo project,
```

Comment:
just for completeness, I would add `PACKIT_DEBUG_DIVIDER` here as well

Reply:
It’s below with the commit-message, it’s a bit confusing though

Reply:
Where did I lose it? 🙃